xml/System.Text.Json/Utf8JsonWriter.xml

```diff
@@ -302,10 +369,12 @@
         <Parameter Name="escape" Type="System.Boolean" />
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
-        <param name="escape">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="value">The UTF-16 encoded value to be written as a UTF-8 transcoded JSON comment.</param>
+        <param name="escape"><see langword="false" /> to indicate the writer should assume the property name is properly escaped and skips the escaping step; otherwise, <see langword="true" />. This is an optional parameter and its default value is <see langword="true" />.</param>
```

Comment:
Same here/below.
```suggestion
        <param name="escape"><see langword="false" /> to indicate the writer should assume the value is properly escaped and skips the escaping step; otherwise, <see langword="true" />. This is an optional parameter and its default value is <see langword="true" />.</param>
```